### PR TITLE
Added extra info about lists (Issue #114)

### PR DESCRIPTION
**Pull request by mish24**
@adammichaelwood I have added some info which I thought was missing in the doc. Are these additions correct? Plus, is there a separate doc needed for these explanations?

### CODE REVIEW

**Comment by adammichaelwood** [comment]
Thanks.

ln 108 -- they are listed, but you can't view deleted forms

ln 134 -- `a directory for each instance.` (not `instances`)

ln 138 `In the case of filled out forms that are not deleted, the data can be reviewed but not edited.`

That is true for Sent, not Deleted, forms, but you have it under the Deleted Forms section.

ln 138, ln 140 `In the case of filled out forms that are...` -- this is too wordy for tech writing. 

 - `Sent Forms can be reviewed but not edited.`
 - `Deleted Forms are listed, but cannot be viewed. They are indicated with the crossed-out eye icon.`

**Comment by mish24** [comment]
@adammichaelwood Actually I was running the emulator today only to check everything on my own. I think all of this is clear to me now. I will make the updates!

**Comment by adammichaelwood** [comment]
Yes -- you should probably never write docs about things you can't actually see or interact with.

**Comment by mish24** [comment]
@adammichaelwood Done. I think if the user is trying out the process as mentioned, this should clear up everything. Is there anything else I can add?

**Comment by mish24** [comment]
screenshot from 2017-09-11 06-39-40

I tried to follow the bullet approach when there is more than a point, like here in the `Tips` section. How do think this should be addressed?
